packages/runtime-vapor/src/component.ts

```diff
@@ -60,6 +61,11 @@ export interface ComponentInternalInstance {
   emitted: Record<string, boolean> | null
   attrs: Data
   refs: Data
+  /**
+   * resolved inheritAttrs options
+   * @internal
+   */
+  inheritAttrs?: boolean
```

Comment:
Is this property really necessary? How about inline it `instance.component.inheritAttrs`